src/update-changelog.ts

```diff
@@ -213,43 +263,17 @@ export async function updateChangelog({
     packageRename,
   });
 
-  // Ensure we have all tags on remote
-  await runCommand('git', ['fetch', '--tags']);
   const mostRecentTag = await getMostRecentTag({
     tagPrefixes,
   });
 
-  const commitRange =
-    mostRecentTag === null ? 'HEAD' : `${mostRecentTag}..HEAD`;
-  const commitsHashesSinceLastRelease = await getCommitHashesInRange(
-    commitRange,
-    projectRootDirectory,
-  );
-  const commits = await getCommits(commitsHashesSinceLastRelease);
-
-  const loggedPrNumbers = getAllLoggedPrNumbers(changelog);
-  const newCommits = commits.filter(
-    ({ prNumber }) =>
-      prNumber === undefined || !loggedPrNumbers.includes(prNumber),
-  );
-
-  const hasUnreleasedChanges =
-    Object.keys(changelog.getUnreleasedChanges()).length !== 0;
-  if (
-    newCommits.length === 0 &&
-    (!isReleaseCandidate || hasUnreleasedChanges)
-  ) {
-    return undefined;
-  }
-
   if (isReleaseCandidate) {
     if (!currentVersion) {
       throw new Error(
         `A version must be specified if 'isReleaseCandidate' is set.`,
       );
     }
-
-    if (mostRecentTag === `${tagPrefixes[0]}${currentVersion ?? ''}`) {
+    if (mostRecentTag === `${tagPrefixes[0]}${currentVersion}`) {
```

Comment:
Hmm, I just realized that we only use the first tag prefix here, which seems strange. This may be a potential bug. But we can investigate that later.

Reply:
The jsdoc does specify that the first prefix is the intended prefix and the rest are fallbacks. `parseChangelog` also only uses the first prefix. 

`getMostRecentTag` does iterate over the tagPrefix list. It should return the used prefix, or we could extract the prefix from the returned most recent tag.

But this also wasn't a change introduced in this PR, so it should get its own ticket.